app/lib/server/functions/updateMessage.js

```diff
@@ -39,8 +40,20 @@ export const updateMessage = function(message, user, originalMessage) {
 		username: user.username,
 	};
 
-	const urls = message.msg.match(/([A-Za-z]{3,9}):\/\/([-;:&=\+\$,\w]+@{1})?([-A-Za-z0-9\.]+)+:?(\d+)?((\/[-\+=!:~%\/\.@\,\w]*)?\??([-\+=&!:;%@\/\.\,\w]+)?(?:#([^\s\)]+))?)?/g) || [];
-	message.urls = urls.map((url) => ({ url }));
+	if (message.parseUrls !== false) {
```

Comment:
I think it would make sense to move this code to a common function and use on `sendMessage` and here at `updateMessage` instead of having this code duplicated, wdyt?

https://github.com/RocketChat/Rocket.Chat/blob/b65372e8062de608d13b1c19464cd12d4ca13cfa/app/lib/server/functions/sendMessage.js#L206

Reply:
Updated!